lib/wallet/rpc.js

```diff
@@ -1735,6 +1755,13 @@ function parseAddress(raw, network) {
     throw new RPCError(errs.INVALID_ADDRESS_OR_KEY, 'Invalid address.');
   }
 }
+function parseDescriptor(raw, network) {
+  try {
+    return Descriptor.fromString(raw, network);
+  } catch (e) {
+    throw new RPCError(errs.INVALID_DESCRIPTOR, 'Invalid descriptor.');
```

Comment:
this is too generic, we should probably point out what's wrong with descriptor
```suggestion
    throw new RPCError(errs.INVALID_DESCRIPTOR, `Invalid descriptor: ${e.message}`);
```

Reply:
Resolved